### PR TITLE
V2

### PR DESCRIPTION
**Pull request by matteosuppo**

### CODE REVIEW

File: demo/app.jsx [hunk 1]
```diff
@@ -173,11 +174,11 @@ class App extends React.Component {
       {device.Name} - IsOpen: <span className={device.IsOpen ? 'open' : 'closed'}>
         {device.IsOpen ? 'true' : 'false'}
       </span> - <a href="#" onClick={(e) => this.handleOpen(e, device.Name)}>
-          open
```
Comment (AlbyIanna) [inline demo/app.jsx]: @matteosuppo pleeease, don't modify the formatting :smile:

**Comment by AlbyIanna** [comment]
looks good to me

File: demo/app.jsx (hunk 1 above)
Comment (matteosuppo) [reply demo/app.jsx]: Sorry vscode decided it was better that way. I didn't even notice. Should I put it the way it was?

Comment (AlbyIanna) [reply demo/app.jsx]: I know, vscode shows its own will sometimes. Yes please :+1:

**Pull request merged**